clients/cobol-lsp-vscode-extension/src/constants.ts

```diff
@@ -26,7 +26,7 @@ export const DOWNLOAD_QUEUE_LOCKED_ERROR_MSG = `Invalid credentials for profile:
 export const UNLOCK_DOWNLOAD_QUEUE_MSG = "Unblock and retry";
 export const INVALID_CREDENTIALS_ERROR_MSG = `Incorrect credentials in Zowe profile ${PROFILE_NAME_PLACEHOLDER}.`;
 export const PROCESS_DOWNLOAD_ERROR_MSG = "Some copybooks could not be located. Ensure your configuration contains correct paths to copybooks, including nested copybooks. Missing copybooks: ";
-export const SPECIFY_DSN_OR_USS = "Please specify DSNs or USS paths for copybooks in settings.";
+export const PROVIDE_PROFILE_MSG = "Please specify a valid Zowe Explorer profile to download copybooks from the mainframe.";
```

Comment:
@Zeb Please check the error message, when the passed profile is not a valid ZE profile

Reply:
that's fine